src/features/events_list/components/ShortState/ShortState.tsx

```diff
@@ -0,0 +1,90 @@
+import { Button, Text } from '@konturio/ui-kit';
+import { useAtom } from '@reatom/react';
+import { useEffect, useState } from 'react';
+import { ErrorMessage } from '~components/ErrorMessage/ErrorMessage';
+import { LoadingSpinner } from '~components/LoadingSpinner/LoadingSpinner';
+import { i18n } from '~core/localization';
+import { currentEventResourceAtom } from '~features/current_event/atoms/currentEventResource';
+import { createStateMap } from '~utils/atoms';
+import { eventListResourceAtom } from '~features/events_list/atoms/eventListResource';
+import { EpisodeTimelineToggle } from '../EpisodeTimelineToggle/EpisodeTimelineToggle';
+import { EventCard } from '../EventCard/EventCard';
+import s from './ShortState.module.css';
+import type { MouseEventHandler} from 'react';
+import type { Event } from '~core/types';
+
+const SingleEventCard = ({
+  event,
+  hasTimeline,
+}: {
+  event: Event;
+  hasTimeline?: boolean;
+}) => (
+  <EventCard
+    event={event}
+    isActive={true}
+    alternativeActionControl={
+      hasTimeline ? <EpisodeTimelineToggle isActive={true} /> : null
+    }
+  />
+);
+
+export function ShortState({
+  hasTimeline,
+  openFullState,
+  currentEventId,
+}: {
+  hasTimeline?: boolean;
+  openFullState: MouseEventHandler<HTMLButtonElement | HTMLDivElement>;
+  currentEventId?: string | null;
+}) {
+  const [event, setEvent] = useState<Event | null>(null);
+  const [{ data: currentEvent, error, loading }] = useAtom(currentEventResourceAtom);
+  const [eventsList] = useAtom(eventListResourceAtom);
+
+  // Try get event from event list
+  useEffect(() => {
+    if (!event && currentEventId && eventsList.data) {
+      const eventFromList = eventsList.data.find((ev) => ev.eventId === currentEventId);
+      if (eventFromList) setEvent(eventFromList);
+    } else if (event && currentEventId === null) setEvent(null);
+  }, [eventsList, currentEventId, setEvent]);
+
+  // Try get event from currentEventResourceAtom
+  useEffect(() => {
```

Comment:
in that case when you switch back to FullState you won't see this event in the list.
is it ok?

Reply:
Good question, answer is - yes, it's okay, but it will be developed soon https://kontur.fibery.io/Tasks/Task/Show-outdated-event-via-link-12519

Reply:
ok, good to know, thank you!